shell/src/main/java/alluxio/shell/command/MakeConsistencyCommand.java

```diff
@@ -0,0 +1,149 @@
+/*
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+ * (the "License"). You may not use this work except in compliance with the License, which is
+ * available at www.apache.org/licenses/LICENSE-2.0
+ *
+ * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied, as more fully set forth in the License.
+ *
+ * See the NOTICE file distributed with this work for information regarding copyright ownership.
+ */
+
+package alluxio.shell.command;
+
+import alluxio.AlluxioURI;
+import alluxio.Constants;
+import alluxio.client.ReadType;
+import alluxio.client.file.FileInStream;
+import alluxio.client.file.FileSystem;
+import alluxio.client.file.FileSystemUtils;
+import alluxio.client.file.URIStatus;
+import alluxio.client.file.options.CheckConsistencyOptions;
+import alluxio.client.file.options.DeleteOptions;
+import alluxio.client.file.options.ListStatusOptions;
+import alluxio.client.file.options.OpenFileOptions;
+import alluxio.exception.AlluxioException;
+import alluxio.wire.LoadMetadataType;
+
+import com.google.common.io.Closer;
+
+import org.apache.commons.cli.CommandLine;
+
+import java.io.IOException;
+import java.util.Collections;
+import java.util.List;
+
+/**
+ * Command for make the consistency of a file between Alluxio and the under storage.
+ */
+public class MakeConsistencyCommand extends AbstractShellCommand {
```

Comment:
another option, since we are already calling checkConsistency as part of this command would be to introduce a flag for the `checkConsistency` command that tries to fix the inconsistencies